js/app.js

```diff
@@ -0,0 +1,53 @@
+// Solicitando un número de tarjeta
+var cardNumber = prompt('Ingrese número de tarjeta:');
+
+// Pasando los caracteres a un nuevo array en orden inverso, conviertiendo de cadena a número
+var reverseArray = function(string) {
```

Comment:
en las instrucciones bajo "consideraciones especificas" dice que tu app.js solo debe tener una funcion "isValidCard"
de cualquier manera, no creo que necesites hacer reverseArray, solo en tu for loop, empieza de atras para adelante.